packages/perseus/src/perseus-api.tsx

```diff
@@ -110,9 +110,6 @@ export const ApiOptions = {
         // Previously handled by `Khan.scratchpad.enable/disable`
         setDrawingAreaAvailable: PropTypes.func,
 
-        // Whether to use the Draft.js editor or the legacy textarea
```

Comment:
Hmm... I should probably remove these as they're not part of the storybook upgrade.